tests/ui/landing_page_test.py

```diff
@@ -24,6 +24,12 @@ def setUp(self):
   def tearDown(self):
     """Teardown for test methods."""
     landing_page = LandingPage(self.driver)
+    try:
+      user_details_button = landing_page.get_element(
```

Comment:
If that's the case, then I would move the try statement down a line.  Also add a comment to explain sometimes click just doesn't work here.

Also, are we sure this is not a real bug, vs webdriver just not working?  i.e. do we know why the click is not working?